models/concerns/validate_timewindows.rb

```diff
@@ -1,43 +0,0 @@
-# Copyright © Mapotempo, 2016
-#
-# This file is part of Mapotempo.
-#
-# Mapotempo is free software. You can redistribute it and/or
-# modify since you respect the terms of the GNU Affero General
-# Public License as published by the Free Software Foundation,
-# either version 3 of the License, or (at your option) any later version.
-#
-# Mapotempo is distributed in the hope that it will be useful, but WITHOUT
-# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
-# or FITNESS FOR A PARTICULAR PURPOSE.  See the Licenses for more details.
-#
-# You should have received a copy of the GNU Affero General Public License
-# along with Mapotempo. If not, see:
-# <http://www.gnu.org/licenses/agpl.html>
-#
-require 'active_support/concern'
-
-module ValidateTimewindows
-  extend ActiveSupport::Concern
-
-  included do
-    validate :validate_timewindows
-  end
-
-  private
-
-  def validate_timewindows
-    t = timewindows.collect{ |timewindow|
-      [timewindow.start, timewindow.end]
-    }
-
-    t.flatten!
-    if t.size > 2 && !t[1..-2].all?
```

Comment:
How do we check now that all only first timewindow has a nil start and the last timewindow has a nil end ?

Reply:
We do not check this now. Is this required ? This was never called before.

Reply:
It is probably not necessary anymore as nil values are converted to integers and timewindows are merged if they overlap